frontend/src/modules/lf/segments/pages/lf-projects-page.vue

```diff
@@ -2,32 +2,46 @@
   <app-page-wrapper>
     <router-link
       v-if="hasPermission(LfPermission.projectGroupEdit)"
-      class="text-gray-600 btn-link--md btn-link--secondary p-0 inline-flex items-center pb-6"
       :to="{
         name: 'adminPanel',
         query: {
           activeTab: 'project-groups',
         },
       }"
     >
-      <lf-icon name="chevron-left" :size="12" class="mr-2" />
-      <span>Project groups</span>
+      <lf-button type="secondary-ghost" class="mb-6">
+        <lf-icon name="angle-left" type="regular" />
+        <span>Project groups</span>
+      </lf-button>
     </router-link>
-    <div class="text-sm text-primary-500 pb-2">
-      {{ projectGroupForm.name }}
-    </div>
 
-    <div class="w-full flex items-center justify-between mb-6">
-      <h4 class="text-gray-900">
-        Manage projects
-      </h4>
-      <el-button
-        v-if="pagination.total && hasPermission(LfPermission.projectCreate) && hasAccessToSegmentId(route.params.id)"
-        class="btn btn--md btn--primary"
-        @click="onAddProject"
-      >
-        Add project
-      </el-button>
+    <div class="w-full flex items-center justify-between mb-6 pt-7 border-t border-gray-100">
```

Comment:
I don't think the rows should have a border. Can you remove?
<img width="1000" alt="Screenshot 2024-12-05 at 12 32 15" src="https://github.com/user-attachments/assets/63574134-30af-4079-9795-5f418dfa7419">

Reply:
done, you've highlighted a different line of code but it's fine I've fixed it.